frontend/src/logic/schedule/shifts-info.logic.ts

```diff
@@ -55,6 +55,7 @@ export class ShiftsInfoLogic implements SectionLogic {
       if (i === null || i === "*") {
         return previousShift || "W";
       }
+      // TODO: What about RPN?
```

Comment:
Jeżeli się nie mylę, w tym samym dniu, kiedy będą harmonogramy

Reply:
Nie do końca rozumiem. Chodziło mi o to, że ten kod jest skracany do RP. Chyba, że nie ma to wpływu na działanie algorytmu to spoko.

Reply:
Dobra. Zapomniałem, o tym że to jest skracane do RP. Więc poprawię to w taki sposób żeby RPN było podmieniane na RP.